tests/test_utils.py

```diff
@@ -52,31 +56,51 @@ def test_best_size_unit_nonmetric2():
     assert unit == 1  # 103bytes == 0.1KB
 
 
-def test_pretty_bytes_metric_fixed1():
-    s = pretty_bytes(1000000, metric=True, precision=0, fixed_unit=2)
+def test_pretty_bytes_fixed_units_metric_fixed1():
```

Comment:
Would it make sense to write a parametrized test to replace all these very similar looking tests?

Reply:
Agreed, I can make that happen.

Reply:
Thanks!